src/neo/SmartContract/NefFile.cs

```diff
@@ -27,7 +27,8 @@ namespace Neo.SmartContract
     │ Magic    │ uint32        │ Magic header                               │
     │ Compiler │ byte[64]      │ Compiler name and version                  │
     ├──────────┼───────────────┼────────────────────────────────────────────┤
-    │ Reserve  │ byte[2]       │ Reserved for future extensions. Must be 0. │
+    │ Source   │ byte[]        │ The url of the source files                │
```

Comment:
Is not better to add a json like this
```json
{
"sourceUrl":[]
}
```
And a create standard for make it easy to expand in the future?

Reply:
We have some reserved fields in NefFile. We can use them first.